plugins/hcp-consul-backend/src/service/router.ts

```diff
@@ -18,6 +18,10 @@ export async function createRouter(
 ): Promise<express.Router> {
   const { config, logger } = options;
 
+  // get version from package.json 
+  const { version } = require('../../package.json');
+  const userAgent = `backstage-plugin-hcp-consul-backend/${version}`
```

Comment:
I feel that the name is a bit long. Should we just name it `backstage-consul/${version}` ?

Reply:
Changed it to backstage-consul-backend since the version is of the backend plugin. This will make it clear.